### ISSUE TITLE
Adding Version/Cloning a language without a version creates 2 versions

### ISSUE THREAD

**Issue by MarcinOkon** [sergeyshushlyapin/Sitecore.FakeDb#119]
I have first noticed it when I was trying to clone an item with two language versions.
English had one version on the clone as expected but the second language had two.

```c#
[Test]
public void CloningVersionsTest()
{
	using (var db = new Db
	{
	  new DbItem("source")
		{
		  new DbField("Title")
			{
			  { "en", "Hello!" },
			  { "de", "Servus!" }
			}
		}
	})
	{
		Item sourceEn = db.GetItem("/sitecore/content/source", "en");
		Item sourceDe = db.GetItem("/sitecore/content/source", "de");

		Assert.AreEqual(1, sourceEn.Versions.Count);
		Assert.AreEqual(1, sourceDe.Versions.Count);

		sourceEn.CloneTo(sourceDe.Parent, "clone", false);
		
		Item cloneEn = db.GetItem("/sitecore/content/clone", "en");
		Item cloneDe = db.GetItem("/sitecore/content/clone", "de");

		Assert.AreEqual(1, cloneEn.Versions.Count);
		Assert.AreEqual(1, cloneDe.Versions.Count); //will fail here with 2 versions
	}
}
```
After further investigations I have found out that adding a version to a language without any version so far seems to have a similar effect and is probably connected

```c#
[Test]
public void AddingVersionTest()
{
	using (var db = new Db
	{
	  new DbItem("home")
	})
	{
		Item homeEn = db.GetItem("/sitecore/content/home", "en");
		Item homeDe = db.GetItem("/sitecore/content/home", "de");

		Assert.AreEqual(1, homeEn.Versions.Count);
		Assert.AreEqual(0, homeDe.Versions.Count);

		homeDe.Versions.AddVersion();

		homeEn = db.GetItem("/sitecore/content/home", "en");
		homeDe = db.GetItem("/sitecore/content/home", "de");

		Assert.AreEqual(1, homeEn.Versions.Count);
		Assert.AreEqual(1, homeDe.Versions.Count); //will fail here with 2 versions
	}
}
```

**Comment by sergeyshushlyapin** [comment]
Looks like a bug. Thanks for reporting!

**Comment by MarcinOkon** [comment]
thank you for looking into this

**Comment by sergeyshushlyapin** [comment]
Released as [v0.36.1](https://www.nuget.org/packages/Sitecore.FakeDb/).
